test/test_utils/sagemaker.py

```diff
@@ -186,7 +186,7 @@ def install_sm_local_dependencies(framework, job_type, image, ec2_conn):
         # TF inference test fail if run as soon as instance boots, even after health check pass. rootcause:
         # sockets?/nginx startup?/?
         print("sleep 300s for tensorflow inference images to avoid socket issues")
```

Comment:
nit: Can u keep this commit as 500s as well.

Reply:
done.